src/striveneditor.css

```diff
@@ -162,15 +162,15 @@
 	background-color: #fff;
 }
 
-.metadata-section, .files-section {
+.Striven-Editor .metadata-section, .files-section {
```

Comment:
.files-section is missing the scope.

Reply:
@sanjogs sharp eyes! Fixed https://github.com/striven-erp/striven-editor/pull/32/commits/82119ece8353b33edb3d42d8b76263c3eaef095d